README.md

```diff
@@ -22,15 +22,15 @@ Function which tries to convert a series of `T`s to `U`s.
 
 A trait that handles base58 encoding and decoding.
 
-## Epoch time
+## epoch_time
 
 Data structure representing time as a `u64`.
 
 ## extend_bytes
```

Comment:
I think this has been removed now

Reply:
It's still there, just removed the chrono conversions